shotover/Cargo.toml

```diff
@@ -33,9 +33,10 @@ kafka = [
     "dep:xxhash-rust",
     "dep:base64",
     "dep:sasl",
+    "dep:serde_json",
 ]
-valkey = ["dep:redis-protocol", "dep:csv", "dep:crc16"]
-opensearch = ["dep:atoi", "dep:http", "dep:httparse"]
+valkey = ["dep:redis-protocol", "dep:csv", "dep:crc16", "dep:serde_json"]
+opensearch = ["dep:atoi", "dep:http", "dep:httparse", "dep:serde_json"]
```

Comment:
This is a good idea, but we should make `serde_json` a non optional dependency instead.
See below `serde_json = { workspace = true, optional = true }`, just remove the `optional` field.